--- a/config.def.h
+++ b/config.def.h
@@ -118,6 +118,13 @@ static const Button buttons[] = {
 	{ ClkStatusText,        0,              Button1,        sigstatusbar,   {.i = 1} },
 	{ ClkStatusText,        0,              Button2,        sigstatusbar,   {.i = 2} },
 	{ ClkStatusText,        0,              Button3,        sigstatusbar,   {.i = 3} },
+	{ ClkStatusText,        0,              Button4,        sigstatusbar,   {.i = 4} },
+	{ ClkStatusText,        0,              Button5,        sigstatusbar,   {.i = 5} },
+	{ ClkStatusText,        MODKEY,         Button1,        sigstatusbar,   {.i = 11} },
+	{ ClkStatusText,        MODKEY,         Button2,        sigstatusbar,   {.i = 12} },
+	{ ClkStatusText,        MODKEY,         Button3,        sigstatusbar,   {.i = 13} },
+	{ ClkStatusText,        MODKEY,         Button4,        sigstatusbar,   {.i = 14} },
+	{ ClkStatusText,        MODKEY,         Button5,        sigstatusbar,   {.i = 15} },
 	{ ClkClientWin,         MODKEY,         Button1,        movemouse,      {0} },
 	{ ClkClientWin,         MODKEY,         Button2,        togglefloating, {0} },
 	{ ClkClientWin,         MODKEY,         Button3,        resizemouse,    {0} },
